fanpy/fanpt/containers/constant_terms.py

```diff
@@ -35,36 +37,41 @@ class FANPTConstantTerms:
     order : int
         Order of the current FANPT iteration.
     previous_responses : np.ndarray
-        Previous responses of the FANPT calculations up to order = order -1
+        Previous responses of the FANPT calculations up to order = order - 1.
+        Shape: (order-1, nactive)
+    quasi_approximation_order : int
+        Either 2 or 3. Controls inclusion of higher-derivative terms.
 
     Methods
     -------
-    __init__(self, fanpt_container, order=1, previous_responses=None)
+    __init__(self, fanpt_container, order=1, previous_responses=None, quasi_approximation_order=2)
         Initialize the constant terms.
     assign_fanpt_container(self, fanpt_container)
-        Assign the FANPT container.
     assign_order(self, order)
-        Assign the order.
     assign_previous_responses(self, previous_responses)
-        Assign the previous responses.
+    assign_quasi_approximation_order(self, qao)
```

Comment:
```suggestion
        Assign the previous responses.
    assign_quasi_approximation_order(self, qao)
```
It would be nice if you could keep the line `Assign the previous responses` as it is a one line explanation for the method `assign_previous_responses`.